dask_jobqueue/lsf.py

```diff
@@ -50,9 +50,10 @@ class LSFCluster(JobQueueCluster):
     # Override class variables
     submit_command = 'bsub <'
     cancel_command = 'bkill'
-    scheduler_name = 'lsf'
 
-    def __init__(self, queue=None, project=None, ncpus=None, mem=None, walltime=None, job_extra=None, **kwargs):
+    def __init__(self, queue=None, project=None, ncpus=None, mem=None, walltime=None, job_extra=None,
+                 configuration='lsf', **kwargs):
+        self.scheduler_name = configuration
```

Comment:
I believe that for clarity, `self.scheduler_name` attribute should be renamed to something else. `self.config_name`?
I know this will mean much more change than this PR is currently doing.

Reply:
I made this change (check notes on most recent commit for more of a sense of scope).